packages/wrapper-react/src/index.tsx

```diff
@@ -30,72 +29,49 @@ export const MonacoEditorReactComp: React.FC<MonacoEditorProps> = (props) => {
     const containerRef = useRef<HTMLDivElement>(null);
 
     useEffect(() => {
-        return () => {
-            destroyMonaco();
+        const destroyMonaco = async () => {
```

Comment:
You can move all methods outside the use effects, so you can use them in either one.

Reply:
See above ⬆️